src/app.js

```diff
@@ -1,9 +1,17 @@
 const express = require('express');
-
+const mongoose = require('mongoose');
+const user = require('./models/users')
 const app = express()
 
-app.post('/signup', (req, res) => {
-    res.send('ok')
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json())
+
+mongoose.connect(process.env.MONGO_URI).catch(error=>console.log(error.message))
+
+app.post('/signup', async(req, res) => {
+    const newuser = await user.create({email:req.body.email,password:req.body.password});
```

Comment:
Your code is not well spaced and organized, a tool named eslint can help do this for you and also force you to follow certain standards. https://dev.to/drsimplegraffiti/eslint-configuration-for-node-project-275l